services/visual-studio-marketplace/visual-studio-marketplace-base.js

```diff
@@ -95,16 +106,14 @@ module.exports = class VisualStudioMarketplaceBase extends BaseJsonService {
 
   transformStatistics({ json }) {
     const { extension } = this.transformExtension({ json })
-    return { statistics: extension.statistics }
-  }
+    const statistics = {}
 
-  getStatistic({ statistics, statisticName }) {
-    for (const statistic of statistics) {
-      if (statistic.statisticName === statisticName) {
-        return { value: statistic.value }
-      }
-    }
+    extension.statistics.forEach(({ statisticName, value }) => {
+      statistics[statisticName] = value
+    })
+
+    const value = validate({ ErrorClass: Error }, statistics, statisticSchema)
```

Comment:
Wasn't sure if the `ErrorClass` specification was required here, but it didn't look like `validate` was setting a default for this